python/ai_service.py

```
@@ -209,46 +188,56 @@ def speech_to_text(self, audio_data: bytes, language: str) -> str:
+                    speech_config=speech_config
+                )
             )
-            
+
+            audio_data = response.candidates[0].content.parts[0].inline_data.data
             logger.info(f"Text to speech ({language}, {character}): Generated audio for '{text[:50]}...'")
-            return response.audio_content
-                
+            return audio_data
+
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)